specs/AppLifecycle/EnvironmentVariables/AppLifecycle - Environment Variables (EV).md

```diff
@@ -310,27 +302,17 @@ packaged, writes to HKCU are virtualized, unless the app has the unvirtualizedRe
 restricted capability and has suppressed registry write virtualization for HKCU.
```

Comment:
Newer versions of the OS allow specific keys to be (un)virtualized, so you can poke a hole just for the environment if you like.